### DISCUSSION TITLE
Some enquiries on optinist usage

### DISCUSSION THREAD

**Discussion by Zywderek** [oist/optinist#508] [Q&A] [answered]
Hiiiii,

Thanks for developing this great GUI to help us with data analysis!

I have some stupid questions after trying your GUI on my two-photon imaging data(in tiff format), it produces a correlation map. But I don't quite understand what it indicates, as there are no x- and y-axis labels in the correlation map. It would help a lot if you would kindly explain a little bit about it.

And one other question is, when I input my Go-NoGo behavior data into GUI for decoding analysis, the system shows the format/shape of neural and behavioral data are not matched. So, I'm wondering how should I organize my behavior training data to make it fit with neural data for decoding analysis.

Deeply appreciate all your support and patience.

Yours sincerely,
Derek

**Comment by ykyamane** [comment, marked as answer]
Hi Derek, 

Correlation function calculates all the pairwise correlation between columns. If the input matrix is m(frames)  x n(number of cells),  it outputs  n x (n -1) number of correlation coefficients.
This is shown in a plot matrix. If you like to calculate the correlation between rows, you can 'on' the 'transpose'  parameter.

Currently, the number of frames of image data and the number of timepoints for the behavior data has to be the same (Otherwise it rejects to calculate). So, you need to preprocess your behavioral data to match the timing of frames of image data.

Best wishes,
Yukako
